tox.ini

```diff
@@ -24,7 +24,7 @@ isolated_build = True
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/requirements-test.txt
 commands =
-  pytest tests/unit tests/meta {posargs}
+  pytest tests/unit {posargs}
```

Comment:
Thought we might as well put them together as we always run them together. Not sure what I had in mind at the time, I thought it would be more like linting jobs or so.